src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -94,6 +94,27 @@ def _wait(self, completions):
             raise Exception([getattr(c, 'error') for c in completions])
 
     def _list_devices(self, cmd):
+        """
+        :returns: Either JSON:
+            [
+              {
+                "name": "sda",
+                "host": "foo",
+                ... lots of stuff from ceph-volume ...
+                "stamp": when this state was refreshed
+              },
+            ]
+
```

Comment:
This function does not currently return JSON. I don't think we ought to document that until it can.

Reply:
Made it clear, it's not yet implemented.